packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.9.0]
+
+### Added
+
+- Add new functions for creating new mock notifications ([#4780](https://github.com/MetaMask/core/pull/4780))
```

Comment:
You may need to expand a bit more on this changelog. Like what specific functions were added